datadog_lambda/logger.py

```diff
@@ -0,0 +1,22 @@
+import logging
+import os
+
+_level_mappping = logging.getLevelNamesMapping()
+_level_mappping.update({
+    # https://docs.datadoghq.com/agent/troubleshooting/debug_mode/?tab=agentv6v7#agent-log-level
+    "TRACE": 5,
+    "WARN": logging.WARNING,
+    "OFF": 100,
+})
+
+def initialize_logging(name):
+    logger = logging.getLogger(name)
+    str_level = os.environ.get("DD_LOG_LEVEL", "INFO").upper() or "INFO"
```

Comment:
Is the `or "INFO"` ever gonna reach? Since you are already setting the default value of the env var to `INFO`?

Reply:
It's to cover the `DD_LOG_LEVEL=""` case. I figured this was easier than adding an `if str_level == ""` later on.

Reply:
Oh, I guess I can remove that first `"INFO"`, that's a good idea.

Reply:
Mmmm, I think the `DD_LOG_LEVEL=""` should be covered by it. We could add it to the tests too, just to confirm.

That also is reached when no env var is set AFAIK.